### PR TITLE
testsuite: Run most of the time with centos/ubuntu normal minions

### PR DESCRIPTION
**Pull request by calancha**
## What does this PR change?
Run most of the time with CentOS and Ubuntu normal minions, instead of running with SSH managed ones.
See https://github.com/SUSE/spacewalk/issues/11972

- testsuite/features/core/allcli_sanity.feature
- testsuite/features/secondary/allcli_config_channel.feature
- testsuite/features/secondary/allcli_overview_systems_details.feature
- testsuite/features/secondary/allcli_reboot.feature
- testsuite/features/secondary/allcli_software_channels.feature
- testsuite/features/secondary/allcli_system_group.feature
- testsuite/features/secondary/min_centos_monitoring.feature
- testsuite/features/secondary/min_centos_salt.feature
- testsuite/features/secondary/min_ubuntu_salt.feature
- testsuite/features/secondary/min_ubuntu_salt_install_package.feature
- testsuite/features/secondary/minssh_centos_salt_install_package_and_patch.feature
- testsuite/features/secondary/srv_group_union_intersection.feature
- testsuite/features/secondary/trad_centos_client.feature

- testsuite/features/init_clients/min_centos_salt.feature
  Rename it from centos_minion.feature
- testsuite/features/init_clients/min_ubuntu_salt.feature:
  Rename it from ubuntu_ssh_minion.feature
- testsuite/features/secondary/min_centos_ssh.feature
  Rename it from min_centos_salt.feature
- testsuite/features/secondary/min_ubuntu_ssh.feature
  Rename from min_ubuntu_salt.feature

- testsuite/run_sets/init_clients.yml
- testsuite/run_sets/refhost.yml
- testsuite/run_sets/secondary.yml
- testsuite/run_sets/sle-updates.yml:
Update with the new file names.

### Ports
- Manager-3.2: https://github.com/SUSE/spacewalk/pull/12017
- Manager-4.0: https://github.com/SUSE/spacewalk/pull/12018
- Manager-4.1: https://github.com/SUSE/spacewalk/pull/12019

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

### CODE REVIEW

File: testsuite/features/secondary/min_centos_salt.feature [hunk 1]
```diff
@@ -65,12 +65,22 @@ Feature: Be able to bootstrap a CentOS minion and do some basic operations on it
...
 
 @centos_minion
-  Scenario: Detect latest Salt changes on the CentOS minion
-    When I query latest Salt changes on "ceos_minion"
+  Scenario: Prepare the SSH-managed CentOS minion
+    Given I am authorized
+    When I enable SUSE Manager tools repositories on "ceos_ssh_minion"
+    And  I enable repository "CentOS-Base" on this "ceos_ssh_minion"
+    And  I install package "hwdata m2crypto wget" on this "ceos_ssh_minion"
+    And  I install package "spacewalk-client-tools spacewalk-check spacewalk-client-setup mgr-daemon mgr-osad mgr-cfg-actions" on this "ceos_ssh_minion"
```
Comment (srbarrios) [inline testsuite/features/secondary/min_centos_salt.feature:73]: For traditional clients sure, but for SSH minion I'm not sure that we need to install those packages.

File: testsuite/features/secondary/min_centos_salt.feature [hunk 2]
```diff
@@ -1,58 +1,58 @@
...
+# 1) delete CentOS minion and register as Centos SSH minion
 # 2) run a remote command
 # 3) try an openscap scan
-# 4) delete CentOS minion client and register as Centos SSH minion
+# 4) delete CentOS SSH minion client and register as Centos minion
 
-Feature: Be able to bootstrap a CentOS minion and do some basic operations on it
+Feature: Bootstrap a SSH-managed CentOS minion and do some basic operations on it
 
 @centos_minion
```
Comment (srbarrios) [inline testsuite/features/secondary/min_centos_salt.feature:11]: We probably need both tags here, centos_minion and centos_ssh_minion

File: testsuite/features/secondary/min_centos_salt.feature [hunk 3]
```diff
@@ -1,58 +1,58 @@
...
-  Scenario: Delete the CentOS SSH minion before normal minion tests
-    When I am on the Systems overview page of this "ceos_ssh_minion"
+  Scenario: Delete the CentOS minion before SSH minion tests
```
Comment (srbarrios) [inline testsuite/features/secondary/min_centos_salt.feature:12]: I wonder why we delete the centos minion, to test centos ssh minion, apparently, we are not reusing the same VM :thinking:

File: testsuite/features/secondary/min_centos_salt.feature [hunk 4]
```diff
@@ -80,23 +90,23 @@ Feature: Be able to bootstrap a CentOS minion and do some basic operations on it
     And I wait until event "OpenSCAP xccdf scanning" is completed
 
 @centos_minion
-  Scenario: Run a remote command on the CentOS minion
+  Scenario: Run a remote command on the SSH-managed CentOS minion
     Given I am authorized as "testing" with password "testing"
     When I follow the left menu "Salt > Remote Commands"
     Then I should see a "Remote Commands" text in the content area
     When I enter command "cat /etc/os-release"
     And I enter target "*centos*"
```
Comment (srbarrios) [inline testsuite/features/secondary/min_centos_salt.feature:98]: Out of scope: Not sure about it, but if that target is looking for the name of the VM, we might want to refactor it, so we pass `"centos_ssh_minion" hostname` as a target. Just thinking in the future if we have someday more VMs matching "*centos*" in this feature's test.

File: testsuite/features/secondary/min_centos_salt.feature (hunk 3 above)
Comment (calancha) [reply testsuite/features/secondary/min_centos_salt.feature:12]: We are using two tags, but currently the VMs are the same, that's why we need to delete in that step.
See here:

`testsuite/features/support/twopence_init.rb`
```
$node_by_host = { 'server'                => $server,
                  'proxy'                 => $proxy,
                  'ceos_minion'           => $ceos_minion,
                  'ceos_ssh_minion'       => $ceos_minion,
                  'ceos_client'           => $ceos_minion,

```

File: testsuite/features/secondary/min_centos_salt.feature (hunk 2 above)
Comment (srbarrios) [reply testsuite/features/secondary/min_centos_salt.feature:11]: Ah nothing sorry, we only have one VM, got confused

File: testsuite/features/secondary/min_centos_salt.feature (hunk 3 above)
Comment (srbarrios) [reply testsuite/features/secondary/min_centos_salt.feature:12]: Cool, got confused, all good then

File: testsuite/run_sets/refhost.yml [hunk 5]
```diff
@@ -20,8 +20,8 @@
 - features/init_clients/sle_client.feature
 - features/init_clients/sle_minion.feature
 - features/init_clients/sle_ssh_minion.feature
-- features/init_clients/centos_ssh_minion.feature
-- features/init_clients/ubuntu_ssh_minion.feature
+- features/init_clients/centos_minion.feature
```
Comment (srbarrios) [inline testsuite/run_sets/refhost.yml:23]: We should also rename both feature files named as min_centos/ubuntu_salt.feature

Comment (calancha) [reply testsuite/run_sets/refhost.yml]: Done.  Renamed as:
- testsuite/features/init_clients/min_centos_salt.feature
- testsuite/features/init_clients/min_ubuntu_salt.feature
- testsuite/features/secondary/min_centos_ssh.feature
- testsuite/features/secondary/min_ubuntu_ssh.feature

**Comment by Bischoff** [comment]
THANKS A LOT. It will be most useful.

**Pull request merged**